datalad/support/sshconnector.py

```diff
@@ -96,9 +101,11 @@ def open(self):
         connection, if it is not there already.
         """
 
+        # set control options
+        ctrl_options = ("-o ControlMaster=auto %s -o ControlPersist=yes" % ' '.join(self.ctrl_options))
```

Comment:
You mean something like `ctrl_options = ["-o ControlMaster=auto", "-o ControlPersist=yes"] + self.ctrl_options` ? this is just being used by open command...